components/definition/cellRules/NetPyNECellRules.js

```diff
@@ -361,179 +418,207 @@ export default class NetPyNECellRules extends React.Component {
-          <div className={"thumbnails"}>
-            <div className="breadcrumb">
-              <FloatingActionButton
-                id={"fromSection2CellRuleButton"}
-                className={"actionButton smallActionButton breadcrumbButton"}
-                style={{ marginTop: "10px", float: "left" }}
-                onClick={() => { that.selectPage("main"); that.setState({ selectedSection: undefined }); }}>
-                {this.state.selectedCellRule}
-              </FloatingActionButton>
-              <div style={{ float: 'left', marginTop: "30px", color: 'grey', fontSize: "20px" }}>&gt;</div>
-              <IconMenu style={{ float: 'left', marginTop: "18px" }}
-                iconButtonElement={
-                  <NetPyNENewSection handleClick={this.handleNewSection} />
+      var mechanismsContainer = Object.keys(mechanismsModel).map( mechName => 
+        <NetPyNEMechanismThumbnail 
+          name={mechName} 
+          key={mechName} 
+          selected={mechName == selectedMechanism} 
+          model={mechanismsModel[mechName]} 
+          deleteMethod={this.deleteMechanism}
+          handleClick={this.selectMechanism} 
+        />
+      )
+    }
+    
+    const content = (
+      <CardText className={"tabContainer"} expandable={true}>
+        <div className={"thumbnails"}>
+          <div className="breadcrumb">
+            <IconButton
+              style={styles.home.container}
+              data-tooltip={ page != "main" ? "Go back to main" : undefined }
+              iconStyle={{color: page != 'main' ? cyan500 : pink400, ...styles.home.icon}}
+              onClick={ () => this.setState({page: 'main', selectedCellRule: undefined, selectedSection: undefined, selectedMechanism: undefined})}
+            >
+              <HomeIcon/>
+            </IconButton>
+
+            <NavigationChevronRight style={styles.rightArrow}/>
+
+            <FloatingActionButton
+              zDepth={1}
+              id="newCellRuleButton"
+              style={styles.cellRule}
+              className={"actionButton smallActionButton"}
+              secondary={ page != 'main' }
+              data-tooltip={ 
+                page != 'main'
+                  ? selectedCellRule && selectedCellRule.length > 8 
+                    ? selectedCellRule 
+                    : "Go back to rule"
+                  : "Create rule"
                 }
-                anchorOrigin={{ horizontal: 'left', vertical: 'top' }}
-                targetOrigin={{ horizontal: 'left', vertical: 'top' }}
-              >
-              </IconMenu>
-            </div>
-            <div style={{ clear: "both" }}></div>
-            {sections}
-          </div>
-          <div className={"details"}>
-            {selectedSection}
-            {dialogPop}
-          </div>
-        </CardText>
-      );
+              onClick={() => this.handleHierarchyClick('main')}
+            >
+              { page != 'main' 
```

Comment:
Can we change this to a classic if/else? Probably you can even extract it to a method and maybe reuse in several places?